origami.py

```diff
@@ -309,11 +309,20 @@ def _on_resize_panes(self, orientation, cells, relevant_index, orig_data, text):
         layout = self._on_resize_panes_get_layout(orientation, cells, relevant_index, orig_data, text)
         self.window.set_layout(layout)
 
-    def zoom_pane(self, fraction):
-        if fraction is None:
-            fraction = .9
+    def zoom_pane(self, auto_zoom):
+        # if fraction is None:
+        #     fraction = .9
+        fraction = .9
+        height = .9
```

Comment:
Use better (consistent) naming now. They both specify same type of unit but one is called fraction and one height.

Reply:
Thanks you for advice, and I clean some comment code, update some settings.